test/integration/kubernetes_deploy_test.rb

```diff
@@ -865,4 +879,8 @@ def test_cronjobs_can_be_deployed
     cronjobs = FixtureSetAssertions::CronJobs.new(@namespace)
     cronjobs.assert_cronjob_present("my-cronjob")
   end
+
+  def test_friendly_error_on_misidentified_erb_file
```

Comment:
This is the new one:

![image](https://user-images.githubusercontent.com/4789493/37627517-2968f36e-2bab-11e8-9312-2e82f00d0542.png)